pkg/virtctl/vm/vm.go

```diff
@@ -88,223 +79,6 @@ var (
 	outputFormat string
```

Comment:
I think you should move also the relevant consts and vars to the files that use them

Reply:
+1

Reply:
Are the `COMMAND_` consts of any use anymore?

Reply:
Some consts are used in multiple files, so these I would keep in this file, the rest could be moved to the file where it belongs to. `COMMAND_` consts are used in tests.